examples/websocket-basic-sample/websocket_basic_sample.md

```diff
@@ -1,6 +1,6 @@
 # WebSocket service - Send/Receive message
 
-This explains how the Ballerina WebSocket server interacts with a WebSocket client. Apart from the `onMessage` remote function given in the example, there are few other remote functions to receive different types of WebSocket messages. The `onOpen` remote function is dispatched as soon as the WebSocket handshake is completed and the connection is established, `onPing` and `onPong` remote methods are dispatched upon receiving ping and pong messages respectively, `onIdleTimeout` remote method is dispatched when the idle timeout is reached, `onClose` is dispatched when a close frame with a statusCode and a reason is received and `onError` is dispatched when an error occurs in the WebSocket connection.
+The `websocket:Service` allows opening up a port via a `websocket:Listener`. A `websocket:Listener` is created by giving the port number, to which `websocket:Service` is attached. The listener accepts and serves connections from WebSocket clients. The `onMessage` remote method is there to receive incoming WebSocket messages. There are a few other remote methods to receive other types of WebSocket messages. The `onOpen` remote method is dispatched as soon as the WebSocket handshake is completed and the connection is established, `onPing` and `onPong` remote methods are dispatched upon receiving ping and pong messages respectively, `onIdleTimeout` remote method is dispatched when the idle timeout is reached, `onClose` is dispatched when a close frame with a `statusCode` and a `reason` is received and finally the `onError` is dispatched when an error occurs in the WebSocket connection. Use this service to implement user applications where you need to establish two-way communication over the WebSocket protocol.
 
```

Comment:
The `onMessage` remote method receives incoming WebSocket messages.

Reply:
Fixed with [7adb6da](https://github.com/ballerina-platform/ballerina-distribution/pull/3748/commits/7adb6da29e1c8db833ffed507f1358d303e7701b)